products/ceqr_survey/models/product/ceqr_survey_bbls.sql

```diff
@@ -1,16 +1,45 @@
-WITH bbls AS (
-    SELECT * FROM
-        {{ ref('stg__pluto') }}
+{% set sql_statement %}
+    SELECT * FROM {{ ref('ceqr_variables') }}
+{% endset %}
+
+{% if execute %}
+    {% set ceqr_variables = run_query(sql_statement) %}
```

Comment:
Not 100% sure why this is needed (the `if execute`). But the compiler isn't happy without it